src/ppx_import.ml

```diff
@@ -81,30 +80,24 @@ let try_find_module_type ~loc env lid =
   (* Here again we prefer to handle the `Not_found` case, so we
      use `Env.lookup_module` rather than `Typetexp.lookup_module`. *)
   try
-    let _path, modtype_decl =
-      Env.lookup_modtype
-        ~loc
-        lid env
-    in
-    Some (match modtype_decl.mtd_type with
-        | None ->
-          raise_errorf ~loc
-            "[%%import]: cannot access the signature of the abstract module %s"
-            (string_of_lid lid)
-        | Some module_type -> module_type)
+    let _path, modtype_decl = Env.lookup_modtype ~loc lid env in
+    Some
+      ( match modtype_decl.mtd_type with
+      | None ->
+        raise_errorf ~loc
+          "[%%import]: cannot access the signature of the abstract module %s"
+          (string_of_lid lid)
+      | Some module_type -> module_type )
   with Not_found -> None
 
 let rec try_open_module_type env module_type =
   match Compat.migrate_module_type module_type with
   | Mty_signature sig_items -> Some sig_items
   | Mty_functor _ -> None
-  | (Mty_ident path | Mty_alias (_, path) ) ->
-    begin match
-        (try Some (Env.find_module path env) with Not_found -> None)
-      with
-      | None -> None
-      | Some module_decl -> try_open_module_type env module_decl.md_type
-    end
+  | Mty_ident path | Mty_alias (_, path) -> (
+    match try Some (Env.find_module path env) with Not_found -> None with
```

Comment:
Minor point: here the formatting is a regression, making the code harder to understand

I think a nice solution would be to rewrite the code using 

```ocaml
match Env.find_module path env
| exception Not_found -> None
| module_decl -> try_open_module_type ...
```

Reply:
I agree that it would make the code easier to read:

```diff
diff --git a/src/ppx_import.ml b/src/ppx_import.ml
index fc93a8b..6cee0f2 100644
--- a/src/ppx_import.ml
+++ b/src/ppx_import.ml
@@ -95,9 +95,9 @@ let rec try_open_module_type env module_type =
   | Mty_signature sig_items -> Some sig_items
   | Mty_functor _ -> None
   | Mty_ident path | Mty_alias (_, path) -> (
-    match try Some (Env.find_module path env) with Not_found -> None with
-    | None -> None
-    | Some module_decl -> try_open_module_type env module_decl.md_type )
+    match Env.find_module path env with
+    | exception Not_found -> None
+    | module_decl -> try_open_module_type env module_decl.md_type )
```

I can make the changes but then it wouldn't be formatting only PR right? Should we address that in another PR instead?

Reply:
As you prefer, given how small the change is, you can add a commit on top of the formatting one and IMHO it'd be fine.